data/json/monster_factions.json

```diff
@@ -348,22 +332,56 @@
   {
     "type": "MONSTER_FACTION",
     "name": "ant",
-    "//": "TODO assign base faction and/or attitude"
+    "base_faction": "insect",
+    "by_mood": [ "zombie", "fungus" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "acid_ant",
-    "//": "TODO assign base faction and/or attitude"
+    "base_faction": "insect",
+    "by_mood": [ "zombie", "fungus" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "crayfish",
+    "base_faction": "insect",
+    "hate": "fish"
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "fungus",
+    "neutral": "bot"
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "jabberwock",
+    "neutral": [ "fish" ],
+    "by_mood": [ "jabberwock" ],
+    "hate": [ "zombie", "animal", "insect", "mutant" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "robofac",
+    "neutral": [ "cop_bot", "defense_bot", "utility_bot", "animal" ],
+    "hate": [ "zombie", "fungus", "cult", "triffid", "nether" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "exodii",
+    "neutral": [ "fish", "small_animal" ],
+    "by_mood": [ "animal", "nether" ],
+    "hate": [ "zombie", "fungus", "cult", "triffid" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "mi-go",
-    "//": "TODO assign base faction and/or attitude"
+    "neutral": [ "fish", "small_animal" ],
+    "hate": [ "zombie", "triffid", "fungus" ]
```

Comment:
Slime?

Reply:
They also got a `nether` BY_MOOD, to keep it obvious, and a reciprocating hate against `exodii` (looters dislike each other).